scripts/install/install-pocs.sh

```diff
@@ -223,7 +223,7 @@ function get_or_build_images {
     if ${DEVELOPER}; then
         echo "Building local PANOPTES docker images."
 
-        cd "${POCS}"
+        cd "${POCS}"                           # TODO: Fix environment variables, 'CD' is not working
```

Comment:
I ended up resolving this by linking the `env` file in `bashrc`.

Reply:
So does the comment need to be here?